ptbr/data.py

```
@@ -0,0 +1,305 @@
+        if text_column == "tokenized_text" and ner_column == "ner":
+            # Spot-check that at least the first record has the expected keys
+            if raw and isinstance(raw[0], dict):
+                if "tokenized_text" not in raw[0] or "ner" not in raw[0]:
+                    available_columns = sorted(raw[0].keys())
+                    available = ", ".join(repr(col) for col in available_columns)
+                    raise ValueError(
+                        f"Data file uses default column mapping but first record "
+                        f"is missing 'tokenized_text' and/or 'ner'. "
+                        f"Available keys: {available}. "
+                        f"Pass text_column= / ner_column= to remap."
+                    )
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)